docs/conf.py

```diff
@@ -29,14 +29,7 @@
 import datetime
 import sys
 
-ON_RTD = os.environ.get('READTHEDOCS') == 'True'
```

Comment:
I think we are still going to need all of this RTD stuff.

Reply:
Should we check if we do need this?

Reply:
See comment below, we are now importing it from sphinx astropy